.travis.yml

```diff
@@ -4,13 +4,14 @@ python:
   - 2.6
   - 2.7
   - 3.5
+cache: pip
 matrix:
   allow_failures:
     - python: 3.5
 install:
+  - pip install -U pip coveralls wheel
   - pip install https://github.com/thijstriemstra/python-messaging/archive/master.zip
-  - pip install coveralls
-  - pip install wheel
+  - pip install -r requirements.txt
```

Comment:
Hm, seems like py3.5 is handling `setup()`s `install_requires` keyword argument slightly differently.  I'm doing some reading on this.